src/Provider/Nominatim/Model/NominatimAddress.php

```diff
@@ -73,24 +73,34 @@ public function withAttribution(string $attribution = null): self
     /**
      * @return null|string
      */
+    public function getCategory()
+    {
+        return $this->category;
+    }
+
     public function getClass()
```

Comment:
We should mark this function as `@deprecated`

Reply:
Done !